scripts/install/install_bmi.sh

```diff
@@ -0,0 +1,62 @@
+set -x
+
+### START BMI setup
+
+TOP_DIR=$(cd $(dirname "$0") && pwd)
+
+# Move to main ims directory for install from script location
+pushd $TOP_DIR/../../
+
+### Install BMI
+
+user=`whoami`
+sudo mkdir -p /var/log/bmi/logs /etc/bmi /opt/bmi
+sudo chown -R $user:$user /etc/bmi /var/log/bmi /opt/bmi
+sudo chmod 775 /etc/bmi /var/log/bmi /opt/bmi
+
+### Copy config/template files
+cp bmi_config.cfg.test /etc/bmi/bmiconfig.cfg
+cp ims/ipxe.temp /etc/bmi/ipxe.config
+cp ims/mac.temp /etc/bmi/pxelinux.cfg
```

Comment:
I meant BMI doesn't pick up the templated from /etc/bmi yet. ipxe.config/cfg pxelinux.cfg/config .